lib/rucio/core/replica.py

```diff
@@ -160,27 +161,32 @@ def __exist_replicas(rse_id, replicas, *, session: "Session"):
     for clause in [path_clause, did_clause]:
         if clause:
             for chunk in chunks(clause, 10):
-                query = session.query(models.RSEFileAssociation.path,
-                                      models.RSEFileAssociation.scope,
-                                      models.RSEFileAssociation.name,
-                                      models.RSEFileAssociation.rse_id,
-                                      models.RSEFileAssociation.bytes,
-                                      func.max(case((models.BadReplicas.state == BadFilesStatus.SUSPICIOUS, 0),
-                                                    (models.BadReplicas.state == BadFilesStatus.BAD, 1),
-                                                    else_=0))).\
-                    with_hint(models.RSEFileAssociation, "INDEX(REPLICAS REPLICAS_PATH_IDX", 'oracle').\
+                stmt = select(
+                    models.RSEFileAssociation.path,
+                    models.RSEFileAssociation.scope,
+                    models.RSEFileAssociation.name,
+                    models.RSEFileAssociation.rse_id,
+                    models.RSEFileAssociation.bytes,
+                    func.max(case((models.BadReplicas.state == BadFilesStatus.SUSPICIOUS, 0),
+                                  (models.BadReplicas.state == BadFilesStatus.BAD, 1),
+                                  else_=0))
+                ).with_hint(models.RSEFileAssociation, "INDEX(REPLICAS REPLICAS_PATH_IDX", 'oracle').\
                     outerjoin(models.BadReplicas,
```

Comment:
Please update the formatting to respect the same formatting rules everywhere. For example: 
```
).with_hint(
  models.RSEFileAssociation, "INDEX(REPLICAS REPLICAS_PATH_IDX", 'oracle'
).outerjoin(
  models.BadReplicas,
```